### PR TITLE
ds category updates

### PR DESCRIPTION
**Pull request by minsa110**
Updated contents for the [data science pages](https://code.visualstudio.com/docs/datascience/overview) as a part of the freshness pass

### CODE REVIEW

File: docs/datascience/jupyter-notebooks.md [hunk 1]
```diff
@@ -278,22 +278,38 @@ If you want to use the full set of debugging features supported in VS Code, such
...
+3. To connect to an existing server for the first time, select **Enter the URL of the running Jupyter server**.
 
-   ![Choose to connect to an existing server](images/jupyter/connect-to-existing.png)
+   ![Choose to connect to an existing server](images/jupyter/select-enter-server-url.png)
 
-1. When prompted to **Enter the URI of a Jupyter server**, provide the server's URI (hostname) with the authentication token included with a `?token=` URL parameter. (If you start the server in the VS Code terminal with an authentication token enabled, the URL with the token typically appears in the terminal output from where you can copy it.) Alternatively, you can specify a username and password after providing the URI.
+4. When prompted to **Enter the URI of a Jupyter server**, provide the server's URI (hostname) with the authentication token included with a `?token=` URL parameter. (If you start the server in the VS Code terminal with an authentication token enabled, the URL with the token typically appears in the terminal output from where you can copy it.) Alternatively, you can specify a username and password after providing the URI.
```
Comment (gregvanl) [inline docs/datascience/jupyter-notebooks.md:311]: Should this be "Enter the URL of the running Jupyter server" to make the prompt?

File: docs/datascience/jupyter-notebooks.md [hunk 2]
```diff
@@ -278,22 +278,38 @@ If you want to use the full set of debugging features supported in VS Code, such
...
+![Search options](images/jupyter/search-filter.png)
+
 ## Connect to a remote Jupyter server
 
 You can offload intensive computation in a Jupyter Notebook to other computers by connecting to a remote Jupyter server. Once connected, code cells run on the remote server rather than the local computer.
 
 To connect to a remote Jupyter server:
 
-1. Select the **Jupyter Server: local** button in the global Status bar or run the **Jupyter: Specify local or remote Jupyter server for connections** command from the Command Palette (`kb(workbench.action.showCommands)`).
+1. Open the Kernel Picker button on the top right-hand side of the notebook (or run the `Notebook: Select Notebook Kernel` command from the Command Palette).
```
Comment (gregvanl) [inline docs/datascience/jupyter-notebooks.md:299]: Please use bold to format commands and UI text. **Notebook: Select Notebook Kernal**

File: docs/datascience/jupyter-notebooks.md [hunk 3]
```diff
@@ -278,22 +278,38 @@ If you want to use the full set of debugging features supported in VS Code, such
 
 ![Debug cell button](images/jupyter/debug-cell.png)
 
+### Search through notebook
+
+You can search (Ctrl / Cmd + F) through an entire notebook or you can search parts of your notebook by filtering your search options. When searching through your notebook, click on the Filter option (funnel icon) to search across:
+- Markdown cell input (`Markdown Source`)
```
Comment (gregvanl) [inline docs/datascience/jupyter-notebooks.md:284]: Use bold for UI text so **Markdown Source**, **Rendered Markdown**, **Code Cell Source**, and **Cell Output**

File: docs/datascience/jupyter-notebooks.md [hunk 4]
```diff
@@ -278,22 +278,38 @@ If you want to use the full set of debugging features supported in VS Code, such
 
 ![Debug cell button](images/jupyter/debug-cell.png)
 
+### Search through notebook
+
+You can search (Ctrl / Cmd + F) through an entire notebook or you can search parts of your notebook by filtering your search options. When searching through your notebook, click on the Filter option (funnel icon) to search across:
```
Comment (gregvanl) [inline docs/datascience/jupyter-notebooks.md:283]: You can use `kbstyle(Ctrl/Cmd + F)` to format like the other keyboard shortcuts

File: docs/datascience/data-science-tutorial.md [hunk 5]
```diff
@@ -117,9 +121,9 @@ This tutorial uses the [Titanic dataset](https://hbiostat.org/data/repo/titanic.
 
    ![Graphing the titanic data](images/data-science-tutorial/jupyter-cell-02.png)
 
-   To better view details on the graphs, you can open them in the plot viewer by hovering over the upper right corner of the graph and clicking the button that appears.
+   > **Tip**: To quickly copy your graph, you can hover over the upper right corner of your graph and click on the `Copy to Clipboard` button that appears. You can also better view details of your graph by clicking the `Expand image` button.
```
Comment (gregvanl) [inline docs/datascience/data-science-tutorial.md:124]: "select the **Copy to Clipboard** button..."
"by selecting the **Expand image** button."

**Comment by gregvanl** [comment]
Looks good. I left a couple of comments and had one question. Otherwise ready to merge and publish

File: docs/datascience/jupyter-notebooks.md (hunk 1 above)
Comment (minsa110) [reply docs/datascience/jupyter-notebooks.md:311]: 👍🏻

**Comment by minsa110** [comment]
Thank you @gregvanl ! Made changes from your comments. Please review again!

**Review by gregvanl — APPROVED** [review]
Thanks!

**Pull request merged**